Marlin/example_configurations/TAZ4/Configuration_adv.h

```diff
@@ -553,14 +553,7 @@
  *
  * Assumption: advance = k * (delta velocity)
  * K=0 means advance disabled.
- * To get a rough start value for calibration, measure your "free filament length"
- * between the hobbed bolt and the nozzle (in cm). Use the formula below that fits
- * your setup, where L is the "free filament length":
- *
- * Filament diameter           |   1.75mm  |    3.0mm   |
- * ----------------------------|-----------|------------|
- * Stiff filament (PLA)        | K=47*L/10 | K=139*L/10 |
- * Softer filament (ABS, nGen) | K=88*L/10 | K=260*L/10 |
+ * See Marlin documentation for calibration instructions.
```

Comment:
Not this one, as it's not valid anymore. If this feature is used by more people in the future and we can get some feedback about used filament (diameter, material), estep/mm setting and calibrated K values we might find a better one.